DIADR.Rproj

```diff
@@ -0,0 +1,18 @@
+Version: 1.0
```

Comment:
@Lilyyyy54 please delete this file and commit the deletion. This shouldn't be here. 

Please also delete `.RData` and `dyadic_trade.RData`.